R/profile_plot.R

```diff
@@ -163,11 +171,13 @@ profile_plot <- function(mydir, rep, para, profilesummary) {
   # parameter vs. final depletion
   plot(x, depl, type = "l", lwd = 2, xlab = label, ylab = "Fraction of unfished", ylim = c(0, 1.2))
   points(est, depl_est, pch = 21, col = "black", bg = "blue", cex = 1.5)
-  abline(h = c(thresh, btarg), lty = c(2, 2), col = c("red", "darkgreen"))
-  legend("bottomright",
-    legend = c("Management target", "Minimum stock size threshold"),
-    lty = 2, col = c("red", "darkgreen"), bty = "n"
-  )
+  abline(h = c(btarg, thresh), lty = c(2, 2), col = c("red", "darkgreen"))
```

Comment:
Did you mean to switch the order of btarg and thresh in the call to `abline()`?

Reply:
I think you did, to match up with the legend labels, but don't you also want to switch the colors? Red seems weird for a target.

Reply:
Good catch. I did this revision for a bunch of runs where I was omitting the lines so had not caught this. Thanks!